src/core/public/injected_metadata/injected_metadata_service.ts

```diff
@@ -108,6 +112,10 @@ export class InjectedMetadataService {
         return this.state.basePath;
       },
 
+      getCategory: () => {
+        return this.state.category;
+      },
```

Comment:
If I understand correctly, the injectedMetadata now returns the category of the currently displayed application, and this is then consumed in the chrome service for it to know which category the displayed app is in, is that right?

If so, this is going to be an issue with NP apps, as they does not reload the page when switching. The information will not be properly updated. This 'currentSection' info should instead be retrieved from the applicationService, that could expose something similar to `currentAppId$` (`currentCategory$` or something)

Reply:
This isn't ever used at the moment - I was just following patterns I was seeing.

How would you feel if I just rid of it?

Reply:
> This isn't ever used at the moment

Is it supposed to be used at some point? If yes, what for? Else it should probably be removed yes.